client/src/components/CommentApp/state/comments.ts

```diff
@@ -85,6 +87,7 @@ export interface Comment {
   author: Author | null;
   date: number;
   text: string;
+  originalText: string;
```

Comment:
Just a minor stylistic point: Might be worth putting ``text``, ``originalText`` and ``newText`` next to each other in the definition and provide a comment that explains why we have three text fields

Reply:
Thanks, done!